### ISSUE TITLE
Weird end of output

### ISSUE THREAD

**Issue by AlexWayfer** [jonathantneal/postcss-extend-rule#1]
Hello!

I have PostCSS config:

```js
module.exports = {
  plugins: [
    require('postcss-extend-rule')
  ]
};
```

And style file:

```scss
button {
  color: red;
}

.button {
  @extend button;
}
```

Output of `postcss test.scss` is:

```css
button {
  color: red;
}

.button {
  color: red;
}

..button {}
```

What is `..button {}`?

Thanks.

**Comment by jonathantneal** [comment]
Resolved by

**Comment by jonathantneal** [comment]
Resolved by #2
